packages/eslint-config-khan/a11y.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable import/no-commonjs */
```

Comment:
This file is mostly copied from the existing config: https://github.com/Khan/wonder-stuff/blob/main/packages/eslint-config-khan/index.js . I'll leave comments in the PR for what's different!